### PR TITLE
ADEN-8511 Change Quantcast label payload

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AnalyticsEngine/AnalyticsProviderQuantServe.php [hunk 1]
```diff
@@ -1,75 +1,84 @@
...
+		$keyValues = F::app()->wg->DartCustomKeyValues;
+		$quantcastLabels = array();
+		$labels = array(
+			'gnre' => 'Genre',
+			'media' => 'Media',
+			'theme' => 'Theme',
+			'tv' => 'TV'
+		);
+
+		if (gettype($keyValues) === 'string') {
```
Comment (fraszczakszymon) [inline extensions/wikia/AnalyticsEngine/AnalyticsProviderQuantServe.php]: What about `is_string($keyValues)`?

File: extensions/wikia/AnalyticsEngine/AnalyticsProviderQuantServe.php [hunk 2]
```diff
@@ -1,75 +1,84 @@
...
+			foreach (explode(';', $keyValues) as $keyValue) {
+				$keyValue = explode('=', $keyValue);
```
Comment (fraszczakszymon) [inline extensions/wikia/AnalyticsEngine/AnalyticsProviderQuantServe.php]:
```suggestion
				list($key, $value) = explode('=', $keyValue);
```
Then `$key`/`$value` will be `string|null`

**Review by fraszczakszymon — APPROVED** [review]

File: extensions/wikia/AnalyticsEngine/AnalyticsProviderQuantServe.php (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AnalyticsEngine/AnalyticsProviderQuantServe.php]: Done.

File: extensions/wikia/AnalyticsEngine/AnalyticsProviderQuantServe.php (hunk 2 above)
Comment (dmnsobczak) [reply extensions/wikia/AnalyticsEngine/AnalyticsProviderQuantServe.php]: Oh yeah, I forgot about that! Changed!

**Pull request merged**